ansible/roles/integration-tests/tasks/open_pull_request.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Create a pull request
+  shell: |
+    curl -s -X POST \
+    -H "Authorization: token {{ ci_pipeline_github_personal_access_token }}" \
```

Comment:
Doesn't this line leak token when using a verbose mode?

Reply:
Good point, I add no_log statement